lib/pages/farm/components/single_logo_farm_title.dart

```diff
@@ -34,11 +34,13 @@ Widget singleLogoFarmTitle(
             ),
           ),
         ),
-        Container(width: 15),
+        Container(width: 100),
```

Comment:
This will cause a large spacing between the token icon and the title. In my opinion it does not look good. Maybe try reducing the space between those to make it look better. Also since you are specifying the width, please use `SizedBox(width: 100)`